pulsar/test_helper.go

```diff
@@ -150,6 +153,12 @@ func deleteTopic(topic string) error {
 	return httpDelete("admin/v2/persistent/" + fmt.Sprintf("%s?force=true", topic))
 }
 
+func createPartitionTopic(t *testing.T, topicName string, partitions int) {
```

Comment:
No need to pass the testing.T here. Let's just return an error and handle the assert in the test.

Reply:
Yes, will do the ut refactor in the future, this pr will remove those.